Blackjack Betting Game

```diff
@@ -109,18 +110,37 @@ def play_game():
         
         # Play again player prompt
         play_cont = 1
+        if player_chips <= 0:
+             print("Game over! You're out of chips.")
         while True:
             choice = input('Do you want to play again? [Y/N]')
             if choice.lower() == 'y':
                 play_cont = 1
                 break
             elif choice.lower() == 'n':
-                print('Thanks for playing!')
+                print (main())
                 play_cont = 0
                 break
             else:
                 print('Wrong Input, Please try again.')
         if play_cont == 0:
             break
             
+def main():
+    print("Welcome to Blackjack!")
+    print("Play Game [Press 1]")
+    print("Quit [Press 2]")
+    while True:
+        choice = input()
+        if choice == "1":
+            play_game()
+        elif choice == "2":
+            print("Thanks for playing!")
+            sys.exit()
```

Comment:
line 139:

Tanggalin mo na to kapag while loop na at dagdagan mo na lang after line 139 ng 'return' function.

Kahit tanggalin mo na din ang 'import sys'.